MAUI/Barcode-Generator/getting-started.md

```diff
@@ -94,11 +94,11 @@ this.Content = barcode;
 
 ![.NET MAUI Barcode Generator Intialize Barcode](images/getting-started/maui-intialize-barcode.png)
 
-N> The default symbology of SfBarcodeGenerator is `Code128`.
+N> The default symbology of SfBarcodeGenerator is [`Code128`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.html).
```

Comment:
Add link for code 128

https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.Code128.html

Reply:
modified sir